components/mpas-ocean/cime_config/buildnml

```diff
@@ -35,9 +35,11 @@ def buildnml(case, caseroot, compname):
     ocn_co2_type      = case.get_value("OCN_CO2_TYPE")
     atm_co2_const_val = case.get_value("CCSM_CO2_PPMV")
     ice_bgc           = case.get_value("MPASI_BGC")
-    nthrds_ocn        = case.get_value("NTHRDS_OCN")
     ocn_pio_typename  = case.get_value("OCN_PIO_TYPENAME")
     ninst_ocn         = 1 # Change if you want multiple instances... though this isn't coded yet.
+    ninst_ocn_real    = case.get_value("NINST_OCN")
```

Comment:
Is there a reason for this to have the _real suffix? More curious than anything....

Reply:
If you look at the line above, ninst_ocn is hardcoded to 1. The build-namelist scripts were using xmlquery to get the real NINST_OCN (not hardcoded to 1).

Reply:
This is probably a flaw in the mpas scripts, with buildnml hardcoding ninst to 1 and build-namelists using the real NINST value, but I didn't want to change behavior.

Reply:
Got it -- thanks.